README.md

```diff
@@ -1,6 +1,7 @@
 # ![Project Link app](https://raw.githubusercontent.com/fxbox/app/master/app/img/icons/32.png "Project Link app") Project Link app
 
 [![Build Status](https://travis-ci.org/fxbox/app.svg?branch=master)](https://travis-ci.org/fxbox/app)
+![Documentation Coverage](https://rawgit.com/fxbox/assets/badge.svg)
```

Comment:
__issue:__ looks like URL is wrong. Would it work if we just use `![Documentation Coverage](./assets/badge.svg)`?